packages/@webex/plugin-meetings/src/meeting/in-meeting-actions.ts

```diff
@@ -99,6 +102,7 @@ export default class InMeetingActions implements IInMeetingActions {
     canRaiseHand: this.canRaiseHand,
     canLowerAllHands: this.canLowerAllHands,
     canLowerSomeoneElsesHand: this.canLowerSomeoneElsesHand,
+    canEndMeeting: this.canEndMeeting,
```

Comment:
issue: This should have related test changes (I think the unit tests are failing due to the changes in here anyway)